packages/fxa-shared/nestjs/health/health.controller.spec.ts

```diff
@@ -1,13 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 import { Test, TestingModule } from '@nestjs/testing';
+
+import { getVersionInfo } from '../version';
 import { HealthController } from './health.controller';
-import { version } from '../version';
+import { HealthControllerConfig } from './health.module';
+
+const version = getVersionInfo(__dirname);
```

Comment:
Woo! We now have a single function we can call from any package to get the version data. Looking forward to removing `version.js/ts` files all over.